src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -114,6 +114,10 @@ bool emberAfGeneralCommissioningClusterCommissioningCompleteCallback(
     app::CommandHandler * commandObj, const app::ConcreteCommandPath & commandPath,
     const Commands::CommissioningComplete::DecodableType & commandData)
 {
```

Comment:
Can you add a comment here - 
```suggestion
// pass fabric and nodeId of commissioner to DeviceControlSvr.
// this allows device to send messages back to commissioner.
// once bindings are implemented, this may no longer be needed. 
```

Reply:
Done.